src/sk_transformers/generic_transformer.py

```diff
@@ -46,7 +135,7 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         Returns:
             pandas.DataFrame: Transformed dataframe.
         """
-        check_ready_to_transform(
+        X = check_ready_to_transform(
```

Comment:
Shouldn't we do this also for `AggregateTransformer`?

Reply:
I added it to all the missing positions.